crates/sqlbuiltins/src/functions/scalars/jq.rs

```diff
@@ -0,0 +1,137 @@
+use std::sync::Arc;
+
+use datafusion::{
+    arrow::datatypes::{DataType, Field},
+    logical_expr::{Expr, ScalarUDF, Signature, TypeSignature, Volatility},
+    physical_plan::ColumnarValue,
+    scalar::ScalarValue,
+};
+use jaq_interpret::{Ctx, FilterT, ParseCtx, RcIter, Val};
+use protogen::metastore::types::catalog::FunctionType;
+use serde_json::Value;
+
+use crate::functions::{BuiltinScalarUDF, ConstBuiltinFunction};
+
+use super::get_nth_scalar_value;
+
+#[derive(Clone)]
+pub struct Jq;
+
+impl ConstBuiltinFunction for Jq {
+    const NAME: &'static str = "jq";
+    const DESCRIPTION: &'static str = "Run a jq query on a JSON document. 
+Note: this uses the rust jaq library, which does not yet support the full jq language.";
```

Comment:
thoughts: should we call this `jaq` so that we don't run into problems if we eventually want to have "real" `jq"?